packages/react/src/index.ts

```diff
@@ -2,11 +2,13 @@ import {
 	useRef,
 	useMemo,
 	useEffect,
-	// @ts-ignore-next-line
-	// eslint-disable-next-line @typescript-eslint/no-unused-vars
-	__SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED as internals,
```

Comment:
This is huge!

Reply:
Before we get too excited, let's remember that it reappeared in https://github.com/preactjs/signals/blob/main/packages/react/runtime/src/auto.ts